client/src/components/ProductDetails/ImageGallery.jsx

```diff
@@ -13,23 +13,66 @@ function ImageGallery ({productStylesArray, currentStyleID}) {
 
   const [currentImage, updateMainImage] = useState(imageGallery[0]);
 
+  const [currentMainImageIndex, updateMainImageIndex] = useState(0);
+
   const setThumbnailClass = (image) => {
     return image === currentImage ? 'imageThumbnailMain' : 'imageThumbnail';
   };
 
 
+  const showNextImage = ()=>{
+    if(currentMainImageIndex !== imageGallery.length-1) {
+      updateMainImageIndex(currentMainImageIndex+1);
+      const newImg = imageGallery.filter((image)=> imageGallery.indexOf(image) === currentMainImageIndex+1);
+      updateMainImage(newImg);
+      // setThumbnailClass(imageGallery.indexOf(currentMainImageIndex+1));
```

Comment:
Is this to update the styling of the selected image? Why is it commented out?

Reply:
I have since decided to do without this function. Deleting now!